configure

```diff
@@ -1008,7 +1016,7 @@ if [[ $USEMKL -eq 1 ]] ; then
   fi
   # Assume GNU linker.
   if [[ $USEOPENMP -eq 1 ]] ; then
-    MKLLIB="-L$MKLROOT $LAPACK $BLAS -Wl,--start-group $MKLROOT/$MKLINTERFACE $MKLROOT/libmkl_intel_thread.a $MKLROOT/libmkl_core.a -Wl,--end-group -liomp5 -lpthread"
+    MKLLIB="-L$MKLROOT $LAPACK $BLAS -Wl, --start-group $MKLROOT/$MKLINTERFACE $MKLROOT/libmkl_intel_thread.a $MKLROOT/libmkl_core.a -Wl,--end-group -liomp5 -lpthread"
```

Comment:
This space should not be here, and was one of the "bugs" I fixed in the Intel compilers when I launched the Jenkins machine.